python/ccf/proposal_generator.py

```diff
@@ -325,6 +325,15 @@ def set_jwt_public_signing_keys(issuer: str, jwks_path: str, **kwargs):
     return build_proposal("set_jwt_public_signing_keys", args, **kwargs)
 
 
+@cli_proposal
+def renew_node_certificate(node_id: str, valid_from: str, valid_to: str, **kwargs):
```

Comment:
Should the proposal name be "trigger_node_certificate_renewal"?

Reply:
Yes!